### ISSUE TITLE
[@nativescript/local-notifications] Payload with schedule notification

### ISSUE THREAD

**Issue by captainhaddockfr35** [NativeScript/plugins#103]
Hi, 

Is it possible to add a payload to a scheduled notification? 
```
LocalNotifications.schedule([
                {
                    id: 1, // generated id if not set
                    title: notification.title,
                    subtitle: notification.body,
                    color: new Color(Config.notificationColor),
                    icon: "res://ic_stat_bulles",
                    ongoing: false, // makes the notification ongoing (Android only)
                    thumbnail: false,
                    //image: "res://bubulle_lightbulb_monocle",
                    channel: "auto-notif", // default: 'Channel'
                    sound: "customsound-ios.wav", // falls back to the default sound on Android
                    at: notification.date, // 10 seconds from now,
                    payload: {
                        data1: "test", 
                        data2: "test"
                    }
                },
            ]).then(
                (scheduledIds) => {
                    console.log('Notification id(s) scheduled: ' + JSON.stringify(scheduledIds));
                },
                (error) => {
                    console.log('scheduling error: ' + error);
                }
            );
```

thank you in advance

**Comment by captainhaddockfr35** [comment]
https://github.com/NativeScript/plugins/pull/111

**Comment by NathanWalker** [comment]
Published support in `5.0.2`
